lib/glimesh_web/live/chat_live/index.ex

```diff
@@ -99,9 +100,33 @@ defmodule GlimeshWeb.ChatLive.Index do
     Chat.list_chat_messages(channel)
   end
 
+  defp list_chat_messages(channel, limit) do
+    # Only currently used when updating elements about the chat.
+    # Just makes sure that if the limit is above 50, it's set back to 50.
+    # Otherwise you could possibly load a few hundred messages when updating elements.
+    # The last 50 should do fine without anyone noticing.
+    # If the viewer's chat currently has less than 50 messages then it just gets however many there are.
+    limit = if limit > 50, do: 50, else: limit
+    Chat.list_chat_messages(channel, limit)
+  end
```

Comment:
What do you mean by this comment? Does it mean that every time a new chat message comes in it refreshes the DOM?

Reply:
Due to the channel_messages list being blank/unordered when reloading all the messages with the timestamp it caused issues. This function will grab up to the last 50 messages from the DB when toggling timestamps.

Reply:
I have to reload the chat when toggling timestamps otherwise they don't show up till the viewer refreshes. Not really sure why since it _is_ a liveview.

Reply:
I think commit ba949c1 completely negates the need for this now as toggling the timestamps actually does what it should now. I believe the temporary assign was causing issues in the first place but it looks like how we handle timeouts is no longer effected by that not being blank.